Tribler/Core/Modules/restapi/channels/base_channels_endpoint.py

```diff
@@ -0,0 +1,76 @@
+import json
+import logging
+import time
+from twisted.web import http, resource
+from Tribler.Core.simpledefs import NTFY_CHANNELCAST
+from Tribler.community.allchannel.community import AllChannelCommunity
+
+
+UNKNOWN_CHANNEL_RESPONSE_MSG = "the channel with the provided cid is not known"
+UNAUTHORIZED_RESPONSE_MSG = "you are not authorized to perform this request"
```

Comment:
Well, this is the error that is passed in the `error` field in the response JSON. It is supposed to be human-readable but there's no clear reason why I didn't use capital letters. At least we are consistent now since we are not using capital letters anywhere else.

Maybe we should work with parseable error messages (such as `unknown_channel`) but this requires a bit more work.